docs/variables/sender.md

```diff
@@ -4,118 +4,220 @@ id: sender
 
 # $(sender)
 
-The `$(sender)` variable represents the user who sent the message.
+Returns a range of metadata around the sender's channel that can be accessed via multiple different variable members for each unique piece of information.
 
-:::caution Timers are not supported.
+:::caution Timers are not supported!
 
-The variable will return `[Error: Cannot be executed inside of a timer]` if you try to use this inside of a timer.
+Timers are executed on an interval by Fossabot itself and therefore this variable cannot be used inside of them.
+
+If your timer iterates through existing commands that use this variable, or if this variable is used in a timer response, Fossabot will return `[Error: Cannot be executed inside of a timer]`.
 
 :::
 
 ## $(sender.accountage)
 
-Returns how long it has been since the sender created their Twitch account.
+Returns the time that has passed since the sender created their **Twitch account**.
+
+#### Parameters
+
+This variable does not take any parameters.
```

Comment:
None of these take parameters, this might be redundant on this page

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/18#discussion_r786586089).